ksql-rest-app/src/test/java/io/confluent/ksql/rest/entity/CommandStatusEntityTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.ksql.rest.entity;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.confluent.ksql.rest.server.computation.CommandId;
+import org.junit.Test;
+
+public class CommandStatusEntityTest {
```

Comment:
I would add deserialization test, e.g. `shouldBeAbleToDeserializeOlderServerMessage` otherwise someone can come along and change the code without realizing _why_ it was like that.

Reply:
Done.